src/server/status/server_status.js

```diff
@@ -2,15 +2,24 @@ import _ from 'lodash';
 
 import states from './states';
 import Status from './status';
+import PluginStatus from './plugin_status';
 
 module.exports = class ServerStatus {
   constructor(server) {
     this.server = server;
     this._created = {};
   }
 
-  create(plugin) {
-    return (this._created[plugin.id] = new Status(plugin, this.server));
+  create(id) {
```

Comment:
Understood: there's no need for a separate `createForService()` method — because there's no need for a separate `ServiceStatus` class that's not going to have any specific properties (unlike for plugins). Agree with this bit.

I guess what's bugging me is that the class hierarchy doesn't match the distinction on the status page. There it looks like there are two distinct kinds of statuses, one for services and one for plugins. Maybe we just need to name the first block of generic statuses on the status page something other than service status, but I'm drawing a blank at the moment. Also this is veering into bikeshedding territory so I'm happy to just go with "service status" if we can't come up with something else soon :)